devicetypes/smartthings/qubino-flush-1d-relay.src/qubino-flush-1d-relay.groovy

```diff
@@ -0,0 +1,378 @@
+/**
+ *  Copyright 2020 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Qubino Flush 1D Relay", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch") {
+		capability "Actuator"
+		capability "Configuration"
+		capability "Health Check"
+		capability "Switch"
+		capability "Refresh"
+		capability "Sensor"
+		
+		fingerprint mfr: "0159 ", prod: "0002", model: "0053", deviceJoinName: "Qubino Flush 1D Relay"
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name: "switch", type: "lighting", width: 6, height: 4, canChangeIcon: true) {
+			tileAttribute("device.switch", key: "PRIMARY_CONTROL") {
+				attributeState "on", label: '${name}', action: "switch.off", icon: "st.switches.switch.on", backgroundColor: "#00A0DC"
+				attributeState "off", label: '${name}', action: "switch.on", icon: "st.switches.switch.off", backgroundColor: "#ffffff"
+			}
+		}
+
+		standardTile("refresh", "device.switch", width: 2, height: 2, inactiveLabel: false, decoration: "flat") {
+			state "default", label: '', action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+
+		main "switch"
+		details(["switch", "refresh"])
+	}
+
+	preferences {
+	parameterMap.each {
+		input (
+			title: it.name,
+			description: it.description,
+			type: "paragraph",
+			element: "paragraph"
+		)
+
+		switch(it.type) {
+			case "boolRange":
```

Comment:
this device has no "boolRange" parameter and a lot of the boilerplate is dedicated to it.

Pare this list down to the actual types included in this PR and also inline most of this code